packages/synapse-react-client/src/SWC.index.ts

```diff
@@ -139,11 +137,9 @@ const SynapseComponents = {
   AccountLevelBadges,
   TermsAndConditions,
   PageProgress,
-  ProjectViewCarousel,
```

Comment:
Was only used in former implementation of SynapseHomepage, was removed a while ago